### PR TITLE
Deposit

### PR DESCRIPTION
**Pull request by CryptoSFor**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,3 @@
+This project implements a Wallet that holds bitcoins. Bitcoin is a type based on float64. 
+Wallet is a type that allows the following operations: to deposit and withdraw money and get
```
Comment (xartishuk) [inline README.md]: newline in the middle of a sentence?

File: wallet.go [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+	balance Bitcoin
+	mutex   sync.Mutex
+}
+
+// Deposit allows depositing bitcoin to the  wallet
+func (w *Wallet) Deposit(b Bitcoin) error {
+	w.mutex.Lock()
+	defer w.mutex.Unlock()
+
+	if math.Signbit(float64(b)) {
```
Comment (xartishuk) [inline wallet.go]: doesn't this work?
```
if float64(b) < 0 {
```

**Review by xartishuk — CHANGES_REQUESTED** [review]
Where are tests?

File: README.md (hunk 1 above)
Comment (CryptoSFor) [reply README.md]: Done

File: wallet.go (hunk 2 above)
Comment (CryptoSFor) [reply wallet.go]: Done

**Review by xartishuk — APPROVED** [review]

**Pull request closed without merging**